### ISSUE TITLE
Find a new ontology for stocks

### ISSUE THREAD

**Issue by dsenalik** [tripal/tripal#1902]
### Task Description

The ontology that we use for entries in the Chado stock table no longer exists.
For example, this would be the correct current URL for one of the terms but returns a 404 error
https://cropontology.org/term/CO_010:0000044

We only use four terms. These are:
```
 accession
 generated germplasm
 cultivar
 414 inbred line
```
DB is "CO_010"
url: http://www.cropontology.org/terms/CO_010:{accession}
urlprefix: http://www.cropontology.org/get-ontology/CO_010
CV is "germplasm_ontology" definition "GCP germplasm ontology"

This task is to determine an appropriate ontology and terms to use instead

### Branch Name

tv4g1-issue1902-new-ontology-for-stock-types

**Comment by laceysanderson** [comment]
Yes, this made me quite unhappy :-( I don't know why crop-ontology just deleted this ontology without consulting anyone 🙈 .

Here are some alternatives:

## accession

This is meant to indicate a generic germplasm accession. We could likely just use the term germplasm here as well. Especially since I also cannot find a good term more specific then that.

- [germplasm (EFO:0007059)](http://www.ebi.ac.uk/efo/EFO_0007059): Germplasm is the living genetic resources such as seeds or tissue that is maintained for the purpose of animal and plant breeding, preservation, and other research uses. These resources may take the form of seed collections stored in seed banks, trees growing in nurseries, animal breeding lines maintained in animal breeding programs or gene banks, etc. Germplasm collections can range from collections of wild species to elite, domesticated breeding lines that have undergone extensive human selection.

## generated germplasm

This was the closest we could get to a generic cross and there are not other good options in the literature :-(

## cultivar

This one is relatively easy to replace. The concept is a cultivated variety.

- [cultivar (EFO:0005136)](https://bioportal.bioontology.org/ontologies/EFO?p=classes&conceptid=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0005136): A cultivated plant variety selected and given a name because it has desirable characteristics that distinguish it from otherwise similar plants of the same species.
- [cultivar epithet (BFO:cultivarEpithet)](https://www.ebi.ac.uk/ols4/ontologies/bco/properties/http%253A%252F%252Frs.tdwg.org%252Fdwc%252Fterms%252FcultivarEpithet?lang=en): Part of the name of a cultivar, cultivar group or grex that follows the scientific name.
- [cultivar (TAXRANK:0000034)](http://purl.obolibrary.org/obo/TAXRANK_0000034): The epithet is usually output in single quotes and may contain multiple words, see ICBN Â§28. Examples: Taxus baccata 'Variegata', Juniperus Ã—pfitzeriana 'Wilhelm Pfitzer'; Magnolia 'Elizabeth' (= a hybrid, no species epithet). 

## 414 inbred line

There is not a good term for this one either :-( The idea was that it would represent an recombinant inbred line (RIL) population.

**Comment by laceysanderson** [comment]
For how to move forward with this, I think we update the default content types but we don't touch the existing terms or any content types someone may have created based on the existing terms.

**Comment by guignonv** [comment]
> I don't know why crop-ontology just deleted this ontology without consulting anyone

I know people managing the CO. I'll ask them why (if don't forget...) and tell you what happened.

**Comment by guignonv** [comment]
OK. No updates were planed for the germplasm ontology and it has been archived, that's why it disappeared. They will try to put redirections so the URLs will still work. I keep you tuned.
EDIT: It was supposed to be archived on [AgroPortal](https://agroportal.lirmm.fr/ontologies) but I could not find it...

**Comment by dsenalik** [comment]
Thanks for following up on this Valentin, much appreciated

**Comment by laceysanderson** [comment]
Yes, very much appreciated! I'm also not able to find it on AgroPortal and have looked multiple times since it disappeared from the crop ontology pages. Might the crop ontology people be able to reach out to AgroPortal in case a misunderstanding happened?

**Comment by laceysanderson** [comment]
I am picking up this task as it is needed for my site. 

## accession

A germplasm accession is a distinct, uniquely identified sample of seeds or plants maintained within a germplasm collection. It represents a specific genetic resource, such as a variety, cultivar, or sample from a wild population. These accessions are crucial for conservation, research, and breeding programs, providing a diverse pool of genetic material

- **[germplasm (EFO:0007059)](http://www.ebi.ac.uk/efo/EFO_0007059): Germplasm is the living genetic resources such as seeds or tissue that is maintained for the purpose of animal and plant breeding, preservation, and other research uses. These resources may take the form of seed collections stored in seed banks, trees growing in nurseries, animal breeding lines maintained in animal breeding programs or gene banks, etc. Germplasm collections can range from collections of wild species to elite, domesticated breeding lines that have undergone extensive human selection.**
- [genetic resource (PBO:0001955)](http://purl.obolibrary.org/obo/PBO_0001955)
- [Organism accession (DATA:2908)](http://edamontology.org/data_2908): An accession of annotation on a (group of) organisms (catalogued in a database).
- [plant resource (PBO:0000808)](https://agroportal.lirmm.fr/ontologies/PBO/PBO_0000808)

## generated germplasm

This was the closest we could get in the past to a generic plant breeding cross.

- **[progeny (PBO:0000065)](https://agroportal.lirmm.fr/ontologies/PBO/PBO_0000065): The offspring of a particular mating.**

## cultivar

This one is relatively easy to replace. The concept is a cultivated variety.

- **[cultivar (EFO:0005136)](https://bioportal.bioontology.org/ontologies/EFO?p=classes&conceptid=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0005136): A cultivated plant variety selected and given a name because it has desirable characteristics that distinguish it from otherwise similar plants of the same species.**
- [cultivar epithet (BFO:cultivarEpithet)](https://www.ebi.ac.uk/ols4/ontologies/bco/properties/http%253A%252F%252Frs.tdwg.org%252Fdwc%252Fterms%252FcultivarEpithet?lang=en): Part of the name of a cultivar, cultivar group or grex that follows the scientific name.
- [cultivar (TAXRANK:0000034)](http://purl.obolibrary.org/obo/TAXRANK_0000034): The epithet is usually output in single quotes and may contain multiple words, see ICBN Â§28. Examples: Taxus baccata 'Variegata', Juniperus Ã—pfitzeriana 'Wilhelm Pfitzer'; Magnolia 'Elizabeth' (= a hybrid, no species epithet). 
- [plant variety (PBO:0000062)](https://agroportal.lirmm.fr/ontologies/PBO/PBO_0000062): A plant differing from other members of the species to which it belongs by the possession of some hereditary traits.

## 414 inbred line

There is not a good term for this one either :-( The idea was that it would represent an recombinant inbred line (RIL) population. Let's just remove support for this content type.

# Germplasm Collection

It would also be extremely helpful to have a content type for the stock collection table. 

- **[collection of organisms (PCO:000000)](https://agroportal.lirmm.fr/ontologies/PCO/PCO_0000000):  May be of the same or different species. A material entity that consists of two or more organisms, viruses, or viroids.**
- [Biospecimen Collection (NCIT:C70945)](http://purl.obolibrary.org/obo/NCIT_C70945): The act or process of gathering biospecimens for testing, diagnostic, propagation, treatment or research purposes.
